core/src/test/kotlin/org/stellar/anchor/filter/Sep10JwtFilterTest.kt

```diff
@@ -21,6 +21,7 @@ import org.stellar.anchor.config.CustodySecretConfig
 import org.stellar.anchor.config.SecretConfig
 import org.stellar.anchor.filter.Sep10JwtFilter.APPLICATION_JSON_VALUE
 import org.stellar.anchor.filter.Sep10JwtFilter.JWT_TOKEN
+import org.stellar.anchor.toSecretKey
 
```

Comment:
Can we add test cases for custodial challenge with bad signatures?

Reply:
It is in Sep10ServiceTests